README.md

```diff
@@ -11,6 +11,7 @@ The server exposes APIs that are consumed by the web app and the mobile app.
 
    - node.js (v12 or newer)
    - mongoDB (v4.2 or newer)
+   - Docker (v19 or newer) for building the image
```

Comment:
this is an optional dependency! mark it as such.

Reply:
Ok, do you want another section with the optional dependencies?